src/classes/DOMMatrix.js

```diff
@@ -182,4 +203,38 @@ export default class DOMMatrix {
       this.m44,
     ]);
   }
+
+  translate(x, y, z) {
+    let translateArray;
```

Comment:
I believe the correct implementation here should check for `arguments.length > 2`. In addtion, we should coerce the arguments to a `Number()` because we don't know if they are strings.

This is pretty typical browser behavior.

Reply:
Additionally, we should probably follow specification:

https://drafts.fxtf.org/geometry/#dom-dommatrixreadonly-translate

Reply:
@jtenner 
Done methinks :)